### ISSUE TITLE
Dashboard sidebar cards don't focus via keyboard navigation

### ISSUE THREAD

**Issue by spmcbride1201** [DecipherNow/gm-fabric-dashboard#493]
Keyboard focus only goes to the button to expand or collapse the dashboard sidebar items. We are unable to actually select a sidebar card itself to navigate to a dashboard.

![image](https://user-images.githubusercontent.com/8700322/31280718-422d2e8a-aa7b-11e7-9587-79ca0ed0f4d2.png)
